<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1091</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Loại xe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại vé</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tồn đầu kỳ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vào trong kỳ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ra trong kỳ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tồn cuối kỳ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Doanh thu</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Từ:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateStart">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>đến:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateEnd">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Loại vé:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbCard">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tất cả loại vé</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vé tháng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vé lượt</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Loại xe:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbVehicle">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tất cả loại xe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Xe máy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ô tô</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Thống kê</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>320</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
